doc/release-notes-1.11.rst

```diff
@@ -71,6 +71,12 @@ Networking
 Bluetooth
 *********
 
+* Multiple fixes to the controller
+* Refactoring of the controller to reduce code running in ISR context
```

Comment:
Delete the point about refactoring. There is ongoing refactoring, but the benefit of the reduced code is not yet realized in the release code base.

Reply:
done